src/services/localStoreService.ts

```diff
@@ -0,0 +1,22 @@
+class LocalStoreService {
```

Comment:
Why did you decide to use `singleton` working with localStorage? In addition - why did you decide to extract working with localStorage into a separate service?

Reply:
first of all, I need something like a facade for communication with `window.localStorage` to simplify work with it. then `window.localStorage` is already singleton so I think it's good idea to have only one instance of it, even if it can't have  his own state

Reply:
I see no sense in creating extra abstraction layer and make it singleton in case the original object is a singleton tbh. Also it seems ok to me to directly use `window.localStorage`. In addition you can also use `localStorage` ref (without window). Try it out in console)

Reply:
Okay, I agree,  localStorageService was deleted.  I know that `localStorage` works without `window` i wrote it in this way to clarify what I'm talking about)